### DISCUSSION TITLE
save per window audio and quality setting in the saved layout

### DISCUSSION THREAD

**Discussion by M0UL** [f1multiviewer/issue-tracker#6] [Feature requests] [answered]
it would be nice if the saved layout remembers  the stream quality and audio settings (volume and audio track)

**Comment by f1multiviewer** [comment, marked as answer]
This will be available in the next release

**Reply by f1multiviewer** [reply]
Stream quality is still an open issue, but the audio settings are now saved!

**Reply by f1multiviewer** [reply]
All done now, some improvements to ensure quality works even if it is slightly different are coming soon.
